client/src/Login.js

```diff
@@ -0,0 +1,81 @@
+import "./Login.css";
+import {
+  getAuth,
+  createUserWithEmailAndPassword,
+  signInWithEmailAndPassword,
+} from "firebase/auth";
+import app from "./firebase.js";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+export const Login = () => {
+  const navigate = useNavigate();
+  const auth = getAuth(app);
+
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  function register() {
+    createUserWithEmailAndPassword(auth, email, password)
+      .then((userCredential) => {
+        const user = userCredential.user;
+        console.log(user);
+        alert("Registered Successfully");
+      })
+      .catch((error) => {
+        const errorCode = error.code;
+        alert(errorCode);
+      });
+  }
+
+  function login() {
+    signInWithEmailAndPassword(auth, email, password)
+      .then((userCredential) => {
+        const user = userCredential.user;
+        user ? navigate("/videos") : alert("Login Failed");
+      })
+      .catch((error) => {
+        const errorCode = error.code;
+        alert(errorCode);
+      });
+  }
+
+  function handleLogin(event) {
+    event.preventDefault();
+  }
+
+  return (
+    <div>
+      <form onSubmit={handleLogin} className="login-form">
+        <legend>Login to your account</legend>
+        <label htmlFor="email" className="login-form-label">
+          Email
+        </label>
+        <input
+          type="email"
+          id="email"
+          onChange={(e) => setEmail(e.target.value)}
+          value={email}
```

Comment:
You could set the input type to `email`, will add useful keys eg to mobile keyboards

Reply:
I think it is currently set to type="email".